src/shared/components/navigation/navigation.component.tsx

```diff
@@ -74,7 +87,13 @@ export default function NavigationComponent(props: Props) {
               </IconButton>
             </Link>
           )}
-
+          <IconButton onClick={handleInfoOpen} className={classes.button}>
+            <InfoTwoTone className={classes.button} />
```

Comment:
```suggestion
            <InfoTwoTone className={classes.button} style={{fontSize: 30}} />
```
I found the info button looked a bit smaller than the GitHub icon, didn’t it?